src/game/editor/mapitems/layer_tune.cpp

```diff
@@ -211,7 +211,7 @@ void CLayerTune::BrushRotate(float Amount)
 	}
 }
 
-void CLayerTune::FillSelection(bool Empty, std::shared_ptr<CLayer> pBrush, CUIRect Rect)
+void CLayerTune::FillSelection(bool Empty, const std::shared_ptr<CLayer> &pBrush, CUIRect Rect)
```

Comment:
```suggestion
void CLayerTune::FillSelection(bool Empty, CLayer *pBrush, CUIRect Rect)
```
Seems like the `shared_ptr` isn't used.

Could you also check the other things where you changed this?